### ISSUE TITLE
Desktop: New notebook and tags buttons missing accessibility labels

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#9251]
## Environment

Joplin version: 2.13.4
Platform: Desktop
OS specifics: Ubuntu 23.10

## Steps to reproduce

1. Enable a screen reader (tested with Orca)
2. Navigate to the "new note" button with <kbd>Tab</kbd>
3. Navigate to the "tags" button with <kbd>Tab</kbd>

**Note**: If the CM6 beta editor is enabled, tab navigation can be used by pressing first <kbd>Escape</kbd>, then <kbd>Tab</kbd>.

## Describe what you expected to happen

The screen reader should read a description of the "new notebook" and "tags" buttons.

**Comment by personalizedrefrigerator** [comment]
When I add an `aria-label` to the "tags" button, the label still isn't read. However, if I then delete the `keyup` handler added by React [to a parent of the `aria-label`, presumably the react root node](https://legacy.reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation), the label *is* read.

I searched for `keyup` event handlers in the `app-desktop` codebase and didn't find any (except perhaps related to TinyMCE).
